jekyll/_cci2/config-intro.adoc

```diff
@@ -366,6 +366,15 @@ In this example, you have:
 
 Using what you have learned above, you are ready to create some powerful pipelines.
 
+[#visual-studio-code-extension]
+## VS Code extension
```

Comment:
Should we include a relevant screenshot/capture here, or just try and keep most images on the overview page?